airbyte-integrations/connectors/destination-postgres/src/main/java/io/airbyte/integrations/destination/postgres/PostgresSQLNameTransformer.java

```diff
@@ -8,6 +8,17 @@
 
 public class PostgresSQLNameTransformer extends StandardNameTransformer {
 
+  // I _think_ overriding these two methods is sufficient to apply the truncation logic everywhere
+  // but this interface + our superclass are weirdly complicated, so plausibly something is missing
+  @Override
+  public String getIdentifier(final String name) {
+    return truncate(super.getIdentifier(name));
```

Comment:
I already made another [mess here](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/destination-postgres/src/main/java/io/airbyte/integrations/destination/postgres/typing_deduping/PostgresSqlGenerator.java#L79) unify that with call to `getIdentifier` ?

Reply:
done in https://github.com/airbytehq/airbyte/pull/36805/commits/8291ba5e1e4432e522e6ea70abfea830b3344fe3.